starboard/raspi/shared/launcher.py

```diff
@@ -64,8 +64,10 @@ class Launcher(abstract_launcher.AbstractLauncher):
   # pexpect times out each second to allow Kill to quickly stop a test run
   _PEXPECT_TIMEOUT = 1
 
+  # Retry Pexpect commands up to this many times
+  _PEXPECT_OUTER_RETRIES = 3
   # Wait up to 30 seconds for the password prompt from the raspi
```

Comment:
Is 30 seconds still relevant here?

Reply:
Yes it's still needed, as it's controlling another retry loop. ( i know, there are too many )